### ISSUE TITLE
Entity state widget shows N/A when brightness is selected for attribute

### ISSUE THREAD

**Issue by dshokouhi** [home-assistant/android#700]
**Home Assistant Android version:**
2.0.0-2-gd747afd-237-full
**Android version:**
10
**Phone model:**
Pixel 4 XL
**Home Assistant version:**
0.113.3
**Last working Home Assistant release (if known):**
N/A
**Description of problem:**

I just setup the new entity state widget and I noticed that when I selected brightness as an attribute to display along with my zwave light status the widget shows as N/A.  If I change the attribute to say the Friendly Name it will display correctly, despite needing a missing space :)

**Traceback (if applicable):**

```

```

**Screenshot of problem:**

Here is what I see when I select brightness as an attribute
![image](https://user-images.githubusercontent.com/1634145/89229453-4ead1c80-d596-11ea-8739-75f019a81184.png)

This is the widget data I used for the above screenshot
![image](https://user-images.githubusercontent.com/1634145/89229491-5c62a200-d596-11ea-9524-87943a08e78f.png)

Here is an example of it working with the friendly name
![image](https://user-images.githubusercontent.com/1634145/89229317-11488f00-d596-11ea-9a90-2c802f31524e.png)

**Additional information:**

**Comment by JBassett** [comment]
Maybe @colincachia can take a look?

**Comment by colincachia** [comment]
> Maybe @colincachia can take a look?

Sure, I can have a look.

@dshokouhi would you mind pasting the value of the brightness attribute for your selected entity?

**Comment by dshokouhi** [comment]
@colincachia sure its for my zwave light switches.  Keep in mind that `brightness` is only displayed when the entity is turned `on`.

Here is the attribute:
`brightness: 255`

For my test as you can see in the 2 screenshots up above that the light was `on` so the attribute should be visible.

**Comment by colincachia** [comment]
> @colincachia sure its for my zwave light switches.  Keep in mind that `brightness` is only displayed when the entity is turned `on`.
> 
> Here is the attribute:
> `brightness: 255`
> 
> For my test as you can see in the 2 screenshots up above that the light was `on` so the attribute should be visible.
> 
> Edit: I also tested it against another zwave light switch with a different brightness level with the same N/A result
> 
> `brightness: 103`

Tapping on the widget should also update the value displayed. Can you confirm that in a scenario where the switch is on (i.e. brightness value should be present), tapping the widget updates the value correctly? 

If that is the case I can simply default to something else than "N/A"

**Comment by dshokouhi** [comment]
> Tapping on the widget should also update the value displayed.

Good to know will update my PR with that tidbit! Otherwise it updates every 30 minutes right? 

> Can you confirm that in a scenario where the switch is on (i.e. brightness value should be present), tapping the widget updates the value correctly?

Unfortunately for me when I tap on the widget the value stays as `N/A` even with the light on

![image](https://user-images.githubusercontent.com/1634145/89231801-e6ad0500-d59a-11ea-9bff-49c752a5b445.png)

Keep in mind also that even the widget label of `test` is missing as well

Here is a side by side comparison of when I select `brightness` or `friendly_name`:

![image](https://user-images.githubusercontent.com/1634145/89231960-41def780-d59b-11ea-9eef-8dad4b6c1081.png)

**Comment by colincachia** [comment]
Thanks for the info, will come up with a fix later on 🙂

**Comment by dshokouhi** [comment]
@colincachia so I noticed that when my light turned off the widget updated on its own but the app was killed.  

![image](https://user-images.githubusercontent.com/1634145/89233995-bfa50200-d59f-11ea-8ede-6d607e831a59.png)


When I launch the app, turn on the light and tap on the widget I see the following error which means I need to start the app again.

![image](https://user-images.githubusercontent.com/1634145/89234035-d21f3b80-d59f-11ea-85fc-ed323401560c.png)

Here is the fatal error I encountered when I grab the latest `logcat`

```
--------- beginning of crash
08-03 15:52:23.840 11125 11125 E AndroidRuntime: FATAL EXCEPTION: main
08-03 15:52:23.840 11125 11125 E AndroidRuntime: Process: io.homeassistant.companion.android, PID: 11125
08-03 15:52:23.840 11125 11125 E AndroidRuntime: java.lang.ClassCastException: java.lang.Integer cannot be cast to java.lang.String
08-03 15:52:23.840 11125 11125 E AndroidRuntime:        at io.homeassistant.companion.android.widgets.StaticWidget.resolveTextToShow(StaticWidget.kt:110)
08-03 15:52:23.840 11125 11125 E AndroidRuntime:        at io.homeassistant.companion.android.widgets.StaticWidget$resolveTextToShow$1.invokeSuspend(Unknown Source:12)
08-03 15:52:23.840 11125 11125 E AndroidRuntime:        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
08-03 15:52:23.840 11125 11125 E AndroidRuntime:        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:56)
08-03 15:52:23.840 11125 11125 E AndroidRuntime:        at android.os.Handler.handleCallback(Handler.java:883)
08-03 15:52:23.840 11125 11125 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:100)
08-03 15:52:23.840 11125 11125 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:214)
08-03 15:52:23.840 11125 11125 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:7356)
08-03 15:52:23.840 11125 11125 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
08-03 15:52:23.840 11125 11125 E AndroidRuntime:        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
08-03 15:52:23.840 11125 11125 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)

```

Let me know if you need anything else pulled but I think this should be helpful to start :)

**Comment by dshokouhi** [comment]
It looks like only the attribute is having an issue.  Just now I tried to set a static widget for my thermostat with the attribute being the `current_temperature` and saw the same N/A error.  When I create a static widget based on the `sensor` state which is numeric it seems to be fine.

**Comment by dshokouhi** [comment]
Fixed in #702
